core/commands/pin.go

```diff
@@ -499,6 +499,9 @@ func pinLsAll(req *cmds.Request, typeStr string, api coreiface.CoreAPI, emit fun
 	}
 
 	for p := range pins {
+		if p.Err() != nil {
+			continue
```

Comment:
@stebalien we could return errors but it doesn't look like we have anything to handle those. No idea if anything upstream it's prepared to actually handle the errors.

This might be easier to sneak into the RC since there won't be any surprises, just fewer issues.

Reply:
I meant, we could just return an error and abort. IIRC, we'll only get one error anyways.

Reply:
That seems like it should work